src/core/StackExchange.Redis.Extensions.Core/Implementations/RedisDatabase.cs

```diff
@@ -58,6 +59,14 @@ public partial class RedisDatabase : IRedisDatabase
         /// <inheritdoc/>
         public ISerializer Serializer { get; }
 
+        /// <summary>
+        /// Utility method for creation of tags
+        /// </summary>
+        /// <param name="tag">The requested tag to set</param>
+        /// <param name="type">The type of the tagged object</param>
+        /// <returns>Tag string</returns>
+        public static string GenerateTagKey(string tag, Type type) => $"tag:{type.Name.GetHashCode()}:{tag}";
```

Comment:
I do not think that it make sense to include this method to public API, consider to create some internal helper for such stuff

Reply:
I agree, did it on auto mode.
Will make private once there is a decision on how to proceed :)